contracts/staking/contracts/src/stake/MixinStakeBalances.sol

```diff
@@ -57,11 +48,11 @@ contract MixinStakeBalances is
     /// @param owner of stake.
     /// @return Active stake for owner.
     function getActiveStake(address owner)
-        public
+        external
```

Comment:
Do we need these getters (`getActiveStake, getInactiveStake, getStakeDelegatedByOwner`)? We don't seem to be using them internally, and while doing a `_loadAndSyncBalance` can be nice, you can get the same information by just looking at the unsynced balance and `currentEpoch`
Optional for this PR, happy to address this in 2151 if we do decide to do this

Reply:
actually after offline discussion with Greg, I think we want to keep these for convenience

Reply:
Yeah, I like keeping these since they load from multiple storage slots in a single call.